src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java

```diff
@@ -71,21 +68,21 @@ private FormFullDTO saveForm(FormRegisterDTO formRegisterDTO) {
 
     }
 
-    private void uploadFile(MultipartFile file, UUID id) {
+    private void uploadFile(MultipartFile multipartFile, UUID id) {
 
         try {
-            Path path = Paths.get(filePath, id.toString());
-            Files.createDirectories(path);
-            byte[] bytes = file.getBytes();
-            BufferedOutputStream stream =
-                    new BufferedOutputStream(new FileOutputStream
-                            (new File(filePath + File.separator + id +
-                                    File.separator + file.getOriginalFilename())));
-            stream.write(bytes);
+
+            byte[] bytes = multipartFile.getBytes();
+
+            File file = new File(filePath + File.separator + id +
+                    File.separator + multipartFile.getOriginalFilename());
+
+            FileUtils.forceMkdirParent(file);
+
+            FileUtils.writeByteArrayToFile(file, bytes);
```

Comment:
variable `bytes` is not used anywhere
```suggestion
            FileUtils.writeByteArrayToFile(file, multipartFile.getBytes());
```

Reply:
i ll marke it in next pr, thanks